index.html

```diff
@@ -12,21 +12,32 @@
 </head>
 
 <body>
-  <header>
-    <div class="header__links">
-      <a class="logo" href="#"><img src="images/MyLogo.png" alt="logo image"></a>
-    </div>
-    <nav>
-      <a class="menu" href="#"><img src="images/humberger.png" alt="hamburger menu"></a>
-      <ul class="menu__list">
-        <li class="menu__list-item"><a href="#projects">Portfolio</a></li>
-        <li class="menu__list-item"><a href="#about-me">About</a></li>
-        <li class="menu__list-item"><a href="#contact">Contact</a></li>
-      </ul>
-    </nav>
-  </header>
+  <div class="body-wrapper">
+    <header>
+      <nav class="navbar">
+        <a href="#home" class="logo"><img src="./images/MyLogo.png" alt="logo"></a>
+
+        <ul class="menu">
+          <li class="nav-item link1">
+            <a href="#projects">Portfolio</a>
+          </li>
+          <li class="nav-item link2">
+            <a href="#about-me">About</a>
+          </li>
+          <li class="nav-item link3">
+            <a href="#contact-form">Contact</a>
```

Comment:
- [ optional] kindly add the font weight and size of the links so they follow the figma design. great job

Reply:
okay, I will arrange it, thank you!